python/letsql/backends/duckdb/__init__.py

```diff
@@ -17,5 +18,16 @@ def execute(
         limit: str | None = "default",
         **_: Any,
     ) -> Any:
-        expr = expr.op().replace(RemoteTableReplacer()).to_expr()
+        expr = expr.op().replace(replace_fix(RemoteTableReplacer())).to_expr()
```

Comment:
unless there's a compelling reason not to, I think all `execute` and `to_pyarrow` definitions should work by way of `to_pyarrow_batches`

Reply:
I agree, solving it in the fix/to_pyarrow_batches PR